src/data/JS/agda-rts.js

```diff
@@ -374,3 +374,18 @@ exports.primShowQName = x => x['name'];
 
 // primQNameFixity : Name -> Fixity
 exports.primQNameFixity = x => x['fixity'];
+
+// Meta
+
+// primShowMeta : Meta -> String
+exports.primShowMeta = x => "_" + x['id'] + "@" + x['module'];
+
+// primMetaToNat : Meta -> Nat
+exports.primMetaToNat = x => x['module'] * 2^64 + x['id'];
```

Comment:
Should this calculation be kept in sync with similar code in other backends or the reflection code?  If yes, please add a comment pointing to the relevant locations?

Reply:
Good point, added.